src/Mvc/Mvc.Api.Analyzers/test/TestFiles/AddResponseTypeAttributeCodeFixProviderIntegrationTest/CodeFixWorksWithValidationProblem.Output.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.AspNetCore.Mvc.Api.Analyzers._OUTPUT_
```

Comment:
We should really update these tests to use the new analyzer infrastructure.

Reply:
Yeah, it might be helpful if we added a list of test areas to https://github.com/dotnet/aspnetcore/issues/36300 and used that to help us push through on that work.